### PR TITLE
Updating Nifi version to 0.4.0

### PR DESCRIPTION
**Pull request by mercul3s**

### CODE REVIEW

**Comment by mercul3s** [comment]
@russellwhitaker please review and merge

File: Berksfile.lock [hunk 1]
```diff
@@ -6,21 +6,22 @@ DEPENDENCIES
...
   ark (0.9.0)
     7-zip (>= 0.0.0)
     windows (>= 0.0.0)
   chef_handler (1.2.0)
-  java (1.35.0)
+  java (1.36.0)
+    apt (>= 0.0.0)
   limits (1.0.0)
-  nifi (0.2.0)
+  nifi (0.3.0)
```
Comment (russellwhitaker) [inline Berksfile.lock]: Shouldn't this be 0.4.0 instead of 0.3.0?

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license          'all_rights'
 description      'Installs/Configures nifi'
 long_description 'Installs/Configures nifi'
-version          '0.2.0'
+version          '0.3.0'
```
Comment (russellwhitaker) [inline metadata.rb]: Also this one: should be 0.4.0 instead of 0.3.0?

**Comment by russellwhitaker** [comment]
Might be a good time to add some README language with this PR.

File: metadata.rb (hunk 2 above)
Comment (mercul3s) [inline metadata.rb]: 0.3.0 is correct, this versioning refers to the cookbook, not the software it's installing.

**Comment by mercul3s** [comment]
Sure, I can add some quick notes. Anything in particular you'd like to see in a README?

File: metadata.rb (hunk 2 above)
Comment (russellwhitaker) [inline metadata.rb]: Ah! GTK, thx

**Comment by russellwhitaker** [comment]
README might indicate purpose of code repo, who uses/maintains, dependencies, how to deploy etc.

**Comment by mercul3s** [comment]
@russellwhitaker updated readme

**Comment by russellwhitaker** [comment]
LGTM

**Pull request merged**